app/components/tournament.js

```diff
@@ -89,6 +89,9 @@ const useStylesFromThemeFunction = createUseStyles(theme => ({
   wrapper: {
     width: '100%',
   },
+  roundTracker: {
+    marginBottom: '59px',
```

Comment:
Please use rem instead of px.  See the [React-Coding-and-Style-Guidelines](https://github.com/EnCiv/.github/wiki/React-Coding-and-Style-Guidelines)

And I will send an offer letter tomorrow.  Thanks!

Reply:
Thanks a lot. I have already converted px to rem.